packages/blade/src/storybook-recipes/SimpleCardWithTokens/Card.native.tsx

```diff
@@ -1,96 +1,17 @@
+/* eslint-disable babel/new-cap */
```

Comment:
Why is this needed 🤔

Reply:
Because I named the style functions with uppercase. 

> A function with a name starting with an uppercase letter should only be used as a constructor.eslint[babel/new-cap](https://eslint.org/docs/rules/new-cap)